src/Bindings/ManualBindings.cpp

```diff
@@ -2968,6 +2969,180 @@ static int tolua_cBlockArea_SaveToSchematicString(lua_State * tolua_S)
 
 
 
+static int tolua_cChatMessageBuilder_AppendPart(lua_State * tolua_S)
+{
+	// function cChatMessageBuilder:AppendPart(Message)
+	// Exported manually to support call-chaining (return *this)
+
+	// Check params:
+	cLuaState L(tolua_S);
+	if (
+		!L.CheckParamUserType(1, "cChatMessageBuilder") ||
+		!L.CheckParamString(2)
```

Comment:
Check param end, too, to avoid weirdnes when calling with more params than needed.